google-cloud-core/src/main/java/com/google/cloud/HttpTransportOptions.java

```diff
@@ -154,10 +156,27 @@ public void initialize(HttpRequest httpRequest) throws IOException {
         if (readTimeout >= 0) {
           httpRequest.setReadTimeout(readTimeout);
         }
+
+        HttpHeaders headers = httpRequest.getHeaders();
+        headers.set("x-goog-api-client", getXGoogApiClientHeader());
       }
     };
   }
 
+  String getXGoogApiClientHeader() {
+    return String.format(
+        "gl-java/%s %s/%s gax/%s",
```

Comment:
C# is reporting it, because they use the GAX library to support their HTTP calls. It really depends on whether Java does the same. If GAX is not used at all for these calls, then remove it.